README.md

```diff
@@ -6,166 +6,1413 @@
 
 [code-style-prettier-image]: https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square
 [code-style-prettier-url]: https://github.com/prettier/prettier
-[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/web3.js.svg?style=flat
-[npm-image]: https://img.shields.io/npm/v/@solana/web3.js.svg?style=flat
-[npm-url]: https://www.npmjs.com/package/@solana/web3.js
+[npm-downloads-image]: https://img.shields.io/npm/dm/@solana/web3.js/preview.svg?style=flat
+[npm-image]: https://img.shields.io/npm/v/@solana/web3.js/preview.svg?style=flat
+[npm-url]: https://www.npmjs.com/package/@solana/web3.js/v/preview
 [semantic-release-image]: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
 [semantic-release-url]: https://github.com/semantic-release/semantic-release
 
 # Solana JavaScript SDK
 
-Use this to interact with accounts and programs on the Solana network through the Solana [JSON RPC API](https://solana.com/docs/rpc).
+This is the JavaScript SDK for building Solana apps for Node, web, and React Native.
```

Comment:
Moved the ‘oh hi welcome here's more information than you expected’ README to the root.